src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -0,0 +1,663 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    [CompilerTrait(CompilerFeature.Patterns)]
+    public class ITuplePatternTests : PatternMatchingTestBase
+    {
```

Comment:
>{ [](start = 4, length = 1)

I'd like to see the IL produced for at least one case. Thanks

Reply:
Not blocking this PR: are we going to expose the ITuple-related information through public API? (ie. what semantic model scenario should be tested?)

---
In reply to: [205633476](https://github.com/dotnet/roslyn/pull/28859#discussion_r205633476) [](ancestors = 205633476)

Reply:
There is no design for semantic model support for pattern-matching for `ITuple`. It is hard to imagine an IDE scenario in which that would be helpful.

---
In reply to: [205633786](https://github.com/dotnet/roslyn/pull/28859#discussion_r205633786) [](ancestors = 205633786,205633476)